filebeat/_meta/common.reference.p2.yml

```diff
@@ -286,6 +286,10 @@ filebeat.inputs:
 # This option is not supported on Windows.
 #filebeat.registry_file_permissions: 0600
 
+# By default Ingest pipelines are not updated. If this option is enabled Filebeat updates
```

Comment:
Maybe clarify to something like: "By default Ingest pipelines are not updated if a pipeline with the same ID already exists."